### PR TITLE
Creates an ARFF Unit and Callsign

### PR DESCRIPTION
**Pull request by Hawaiicoder**
Addresses #1554

The purpose of this pull request is to create a callsign for ARFF operations.

### CODE REVIEW

**Comment by Hawaiicoder** [comment]
Build is failing, even though I ran it through a JSON Linter. What's wrong?

**Comment by JPmAn24** [comment]
Let me take a look at this real quick

**Comment by JPmAn24** [comment]
@Hawaiicoder Are you part of our slack? 

If not, here's a link: http://slack.openscope.io/

**Comment by Hawaiicoder** [comment]
Still can't find the issue. I bet it's something dumb on my part, though.

File: assets/airlines/ARFF [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "icao": "null",
+    "name": "Airport Rescue and Firefighting",
+    "callsign": {
+        "name": "Rescue",
+        "callsignFormats": [
+            "Rescue #",
```
Comment (n8rzz) [inline assets/airlines/ARFF:7]: the trailing comma here is an issue

File: assets/airlines/ARFF [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+        ]
+    },
+    "fleets": {
+        "default": [
+            ["ARFF", 10],
```
Comment (n8rzz) [inline assets/airlines/ARFF:13]: the trailing comma here is an issue

File: assets/airlines/ARFF [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+        ],
```
Comment (n8rzz) [inline assets/airlines/ARFF:15]: the trailing comma here is an issue

File: assets/airlines/ARFF (hunk 1 above)
Comment (Hawaiicoder) [reply assets/airlines/ARFF:7]: Which line? 5 or 7?

**Comment by Hawaiicoder** [comment]
Thank you so much for your help!

**Comment by erikquinn** [comment]
@hawaiicoder can you provide a reference to something that shows this callsign in use real world to verify that adding this would be realistic?

**Comment by Hawaiicoder** [comment]
@erikquinn Well, it's a bit more of a generic callsign, most airports use different names. Most common from what I've heard is Truck 1, or Emergency Truck 1. I figured that we should use something a bit more generic to avoid confusion. 

I'm also sort of stuck. I'm not sure how to get aircraft to declare emergencies by themselves and how to be able to only call up ARFF when it's needed for an emergency inbound. any advice is appreciated.

**Pull request closed without merging**